web3.js/src/system-program.ts

```diff
@@ -670,7 +690,7 @@ export class SystemProgram {
   ): TransactionInstruction {
     let data;
     let keys;
-    if (params.basePubkey) {
+    if ('basePubkey' in params) {
```

Comment:
Minor nit: maybe prettier should convert single quotes to doubles like in our React projects (for extra consistency).